src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BaseTemplatedView.shared.cs

```diff
@@ -24,15 +24,15 @@ protected override void OnBindingContextChanged()
 			base.OnBindingContextChanged();
 
 			if (control != null)
-				Control.BindingContext = BindingContext;
+				control.BindingContext = BindingContext;
```

Comment:
doesn't make more sense to use the prop. instead of the field?

Reply:
Agreed.
Updated